backend/src/main/java/de/jankahle/capstone/controller/RecipeController.java

```diff
@@ -25,4 +24,9 @@ public RecipeController(RecipeService recipeService) {
     public Recipe saveRecipe(@RequestBody RecipeDto recipeDto) {
         return recipeService.saveRecipeToDB(recipeDto);
     }
+
+    @GetMapping
+    public List<Recipe> loadRecipes() {
+        return recipeService.loadRecipesFromDB();
```

Comment:
i would not specify that you load the Recipe from the db. This is not relevant for the functionality and could be miss leading when you change the functionality